package.json

```diff
@@ -38,6 +38,7 @@
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^4.0.0",
     "eslint-plugin-react": "^6.10.3",
+    "js-yaml": "^3.9.1",
```

Comment:
I'm not opposed to this for testing, but you could also just provide a mock parse function and verify it gets called and what it returns is used instead.